editor/src/viewport_tools/snapping.rs

```diff
@@ -249,22 +249,22 @@ impl SnapHandler {
 	/// Add the control points (optionally including bézier handles) of the specified shape layer to the snapping points
```

Comment:
What is meant by "control points" here? Was that supposed to mean ManipulatorGroup or something? The actual meaning of a control point is a synonym for a handle, but we prefer to use handle. But I don't think "handles" is the right word here because of what's included in the parenthesis right after it.